app/renderer/components/main/main.js

```diff
@@ -402,12 +384,11 @@ class Main extends ImmutableComponent {
       windowActions.setDownloadsToolbarVisible(false)
     })
 
-    const self = this
     ipc.on(messages.BLOCKED_RESOURCE, (e, blockType, details) => {
-      const frameProps = frameStateUtil.getFrameByTabId(self.props.windowState, details.tabId)
-      frameProps && windowActions.setBlockedBy(frameProps, blockType, details.url)
+      windowActions.setBlockedBy(this.props.activeFrameKey, blockType, details.url)
     })
 
+    // TODO can we remove this?
     ipc.on(messages.BLOCKED_PAGE, (e, blockType, details) => {
```

Comment:
So right now we have only comment in this call. We need to determinate what to do with it. Currently we are not doing anything. Now if you don't have any objections I will remove it otherwise we need to add some logic into this function. In this state it's waste of space :smiley: